src/Directory.Build.props

```diff
@@ -4,24 +4,69 @@
   <!--
     Add any shared properties you want for the projects under this directory that need to be set before the auto imported Directory.Build.props
   -->
-  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory).., Directory.Build.props))\Directory.Build.props" />
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory).., Directory.Build.props))\directory.build.props" />
```

Comment:
Shouldnt this be Directory.Build.props - casing is very important for msbuild on *NIX

Reply:
It is indeed important. I just thought we were going lowercase on these, but can see the file in question is actually still in title case.